docs/ref/settings.txt

```diff
@@ -2006,6 +2006,13 @@ Example: ``"http://media.example.com/"``
     :setting:`MEDIA_URL` and :setting:`STATIC_URL` must have different
     values. See :setting:`MEDIA_ROOT` for more details.
 
+.. note::
+
+    If :setting:`MEDIA_URL` is a relative path (i.e., not a URL), then the
+    ``SCRIPT_NAME`` header's value (or ``/`` if the header is not set) will be
```

Comment:
`SCRIPT_NAME` is not a header but rather a wsgi environment value.

Reply:
done